sketchlib/prevent_mobile_scroll.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Based on https://kirkdev.blogspot.com/2020/10/prevent-browser-scrolling-while-drawing.html
+ * which in turn is based on https://stackoverflow.com/questions/49854201/html5-issue-canvas-scrolling-when-interacting-dragging-on-ios-11-3/51652248#51652248
+ * @param {HTMLCanvasElement} canvas The canvas to use
+ */
+export function prevent_mobile_scroll(canvas) {
+  const callback = (event) => {
+    if (event.target === canvas) {
+      event.preventDefault();
+    }
+  };
+
+  const options = { passive: false };
```

Comment:
Mention that this option is needed so the browser back gesture is also deactivated.